hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/SyntheticCryptoTransferApprovalsMigrationTest.java

```diff
@@ -186,6 +211,181 @@ void repeatableMigration() {
+        // given
+        domainBuilder
+                .recordFile()
+                .customize(r -> r.hapiVersionMajor(0).hapiVersionMinor(39).hapiVersionPatch(0))
+                .persist();
+        // given
+        var contractId = EntityId.of("0.0.2119900", CONTRACT);
+        var contractId2 = EntityId.of("0.0.2119901", CONTRACT);
+        var priorContractId = EntityId.of("0.0.2119899", CONTRACT);
+        var currentKeyUnaffectedEntity = entityCurrentKey(contractId.getEntityNum());
+        var currentKeyAffectedEntity = entityCurrentKey(contractId2.getEntityNum());
+        var noKeyEntity = entityWithNoKey();
+        var pastKeyUnaffectedEntity = entityPastKey(contractId.getEntityNum());
+        var pastKeyAffectedEntity = entityPastKey(contractId2.getEntityNum());
+        var thresholdTwoKeyEntity = domainBuilder
+                .entity()
+                .customize(e -> e.key(getThresholdTwoKey(contractId.getEntityNum()))
+                        .timestampRange(Range.atLeast(getTimestampWithinBoundary()))
+                        .build())
+                .persist();
+
+        var cryptoTransfersPair = setupCryptoTransfers(
+                contractId,
+                contractId2,
+                priorContractId,
+                currentKeyUnaffectedEntity,
+                currentKeyAffectedEntity,
+                noKeyEntity,
+                pastKeyUnaffectedEntity,
+                pastKeyAffectedEntity,
+                thresholdTwoKeyEntity);
+
+        var nftPastKeyUnaffectedEntity = entityPastKey(contractId.getEntityNum());
+        var nftPastKeyAffectedEntity = entityPastKey(contractId2.getEntityNum());
+        var nftTransfersTransactionPair = setupTransactionNfts(
+                contractId,
+                priorContractId,
+                currentKeyUnaffectedEntity,
+                currentKeyAffectedEntity,
+                noKeyEntity,
+                nftPastKeyUnaffectedEntity,
+                nftPastKeyAffectedEntity,
+                thresholdTwoKeyEntity);
+
+        var tokenPastKeyUnaffectedEntity = entityPastKey(contractId.getEntityNum());
+        var tokenPastKeyAffectedEntity = entityPastKey(contractId2.getEntityNum());
+        var tokenTransfersPair = setupTokenTransfers(
+                contractId,
+                contractId2,
+                priorContractId,
+                currentKeyUnaffectedEntity,
+                currentKeyAffectedEntity,
+                noKeyEntity,
+                tokenPastKeyUnaffectedEntity,
+                tokenPastKeyAffectedEntity,
+                thresholdTwoKeyEntity);
+
+        // when
+        migration.onEnd(RECORD_FILE);
+
+        // then
+        var expectedCryptoTransfers = new ArrayList<>(cryptoTransfersPair.getLeft());
+        expectedCryptoTransfers.addAll(cryptoTransfersPair.getRight());
+        assertThat(cryptoTransferRepository.findAll()).containsExactlyInAnyOrderElementsOf(expectedCryptoTransfers);
```

Comment:
Use the more succinct `List.of()`. Same comment for other assertions.

```suggestion
assertThat(cryptoTransferRepository.findAll()).containsExactlyInAnyOrderElementsOf(List.of(cryptoTransfersPair.getLeft(), cryptoTransfersPair.getRight()));
```